lib/logging.js

```diff
@@ -52,19 +54,41 @@ module.exports = function(logger) {
     return req.ip;
   }
 
+  /**
+   * Check if the request URL starts with any of the blacklist paths.
+   *
+   * @param {String} url
+   *    Request url
+   * @return {Boolean}
+   *    True if the request URL is included in the blacklist.
+   */
+  function isUrlBlackedList(url) {
+    if (blacklist) {
```

Comment:
Minor: Consider using a more functional approach to reduce execution branches
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some
```js
// not tested
blacklist = blackist || [];
function isUrlBlackedList(url) {
   return blackist.some(url.startsWith);
}
```